src/frontend/screens/Game/GamePage/components/LaunchOptions.tsx

```diff
@@ -54,7 +54,6 @@ const LaunchOptions = ({ gameInfo, setLaunchArguments }: Props) => {
         }
       }}
       value={selectedLaunchOptionIndex.toString()}
-      prompt={t('launch.options', 'Launch Options...')}
```

Comment:
Removed this to avoid showing an empty selection. It will show directly the first option now.